docs/deployment-examples/sql-server-databases.md

```diff
@@ -1,48 +1,91 @@
+
+Most of the tooling from above requires it to be installed somewhere.  It is important the same version is used across all environments.  This prevents any uncertainty during deployments.  
+
+Finally, it is good security practice to have a different account with schema change permissions per environment.  An account used to change a test environment should not be able to change production.  
+
+With all that in mind, a "jump box" is where tentacles should be installed.  The jump box sits between Octopus Deploy and the SQL Server VIP.  The tentacle is running as a [service account](/docs/infrastructure/windows-targets/running-tentacle-under-a-specific-user-account.md) with the necessary permissions to make schema changes.  The tooling chosen for database deployments is installed on the jump box.
+
+![](database-with-jump-box.png "width=500") 
+
+In the event of multiple domains, a jumpbox would be needed per domain.  This might be seen where there is a domain in a local infrastructure and another domain in a cloud provider such as Azure.  As long as port 10933 is open (for a listening tentacle) or port 443 (for a polling tentacle) Octopus will be able to communicate to the jumpbox.
+
+![](database-jump-box-multiple-domains.png "width=500")
+
+It is possible to install many tentacles on a single server.  Please [read here](/docs/administration/managing-multiple-instances.md) for more information.  
+
+![](database-jump-box-multiple-tentacles.png "width=500")
+
+## Database Deployment Permissions {#SQLServerdatabases-Permissions}
+
+The level of permissions required to automate database deployments is tricky.  There is a fine line between functionality and security.  There is no single magic bullet.  It will be up to you and your security team to discuss.  With that said, below are some considerations around permissions and a couple of recommendations.
+
+### Deployment Permission Considerations {#SQLServerdatabases-DeploymentPermissions}
+
+The account used to make schema changes requires elevated permissions.  Because of that, create a special account to handle database deployments.  Do not use the same account used by an IIS Web Application.
+
+The level of elevated permissions is up to you.   The more restrictions placed on the deployment account means more manual steps.  Deployments will fail due to missing or restricted permissions.  Octopus will provide the error message to fix the issue.  It will need a manual intervention.  It is up to you to decide which is best. 
+
+First, decide what the deployment account should have the ability to do at the server level.  From there, research which server roles can are applicable.  For example, the account can create databases and users.  Use the securityadmin and dbcreator server roles.  Should the account only be able to create databases? Use dbcreator role. 
```

Comment:
First, decide what the deployment account should have the ability to do at the server level.  From there, research which server roles ~~can~~ are applicable.

Reply:
I'm surprised both HemingwayApp and Grammarly didn't catch that.  Good find.